epuck_ros2_cpp/include/epuck_ros2_cpp/i2c_wrapper.hpp

```diff
@@ -32,6 +32,26 @@ class I2CWrapper {
   virtual int setAddress(int address) = 0;
   virtual int readData(char *buffer, int size) = 0;
   virtual int writeData(char *buffer, int size) = 0;
+  int readRegister(char reg, char *data, int size) {
+    if (this->writeData(&reg, 1) != 1)
+      return -1;
+    if (this->readData(data, size) != size)
+      return -2;
+    return 0;
+  }
+  char readInt8Register(char reg) {
+    int status;
+    char byte;
+
+    for (int i = 0; i < 3; i++) {
```

Comment:
A small comment explaining why you are doing a loop would be welcome

Reply:
I added `mMaxRetryCount` instead of the magic number, it may be more readable than a comment